File: src/main/java/top/litop/ptgame/MainVerticle.java
package top.litop.ptgame;

import io.vertx.core.AbstractVerticle;

public class MainVerticle extends AbstractVerticle {
    @Override
    public void start() throws Exception {
        vertx.deployVerticle("top.litop.ptgame.room.MainService");
    }

    @Override
    public void stop() throws Exception {

    }
}

File: ptgameback/ptgameback/src/main/kotlin/top/litop/ptgameback/MainVerticle.kt
package top.litop.ptgameback

import io.vertx.core.AbstractVerticle

class MainVerticle : AbstractVerticle() {
  override fun start(){
    vertx.deployVerticle("top.litop.ptgameback.room.MainService")
  }
  override fun stop(){

  }
}
